app/services/ratings-service.server.ts

```diff
@@ -0,0 +1,25 @@
+import { BaseService } from "~/services/base-service.server";
+import type { Rating } from "@prisma/client";
+import { db } from "~/utils/db.server";
+
+export class RatingsService extends BaseService {
+  public upsertRating(rating: { jokeId: number; score: number; ratingId?: number }): Promise<Rating> {
+    return db.rating.upsert({
```

Comment:
used an upsert so that we can easily change ratings without needing a separate operation